config/jobs/kubernetes-sigs/downloadkubernetes/downloadkubernetes-presubmits.yaml

```diff
@@ -0,0 +1,18 @@
+presubmits:
+  kubernetes-sigs/downloadkubernetes:
+  - name: pull-downloadkubernetes-verify-index
+    always_run: true
+    optional: false
+    decorate: true
+    path_alias: "sigs.k8s.io/downloadkubernetes"
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20201208-8b354d9-master
```

Comment:
Use a lighterweight image. Compare presubmits in the other RelEng repos for guidance.

Reply:
done